src/main/java/mate/academy/dao/BookDaoImpl.java

```diff
@@ -0,0 +1,129 @@
+package mate.academy.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.ConnectionUtil;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Book;
+
+@Dao
+public class BookDaoImpl implements BookDao {
+
+    @Override
+    public Book create(Book book) {
+        String query = "INSERT INTO books (title,price) VALUES(?,?)";
```

Comment:
Maybe all queries we need move to constants, but I don't sure.

Reply:
Also you can rename `query` to `insertQuery` for exsample

Reply:
Rename to instertQuery

Check and rename all queries variables